dspace-server-webapp/src/test/java/org/dspace/app/rest/signposting/controller/LinksetRestControllerIT.java

```diff
@@ -674,4 +798,111 @@ public void findTypedLinkForUnDiscoverableItem() throws Exception {
         choiceAuthorityService.clearCache();
     }
 
+    @Test
+    public void getDescribedBy() throws Exception {
+        context.turnOffAuthorisationSystem();
+        DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+        String currentDateInFormat = dateFormat.format(new Date());
+        String title = "Item Test";
+        Item item = ItemBuilder.createItem(context, collection)
+                .withTitle(title)
+                .withMetadata("dc", "identifier", "doi", doi)
+                .build();
+        String responseMimeType = configurationService.getProperty("signposting.describedby.mime-type");
+        context.restoreAuthSystemState();
+
+        String titleXml = "<title xmlns=\"http://datacite.org/schema/kernel-3\">" + title + "</title>";
+        String doiXml = "<alternateIdentifier xmlns=\"http://datacite.org/schema/kernel-3\" " +
+                "alternateIdentifierType=\"doi\">" + doi + "</alternateIdentifier>";
+        String handleXml = "<alternateIdentifier xmlns=\"http://datacite.org/schema/kernel-3\" " +
+                "alternateIdentifierType=\"uri\">http://localhost:4000/handle/" + item.getHandle() +
```

Comment:
I suggest to simplify this test, it is not its responsibility to check for the correctness of the xslt result.
We should only check that
- the response has the right mimetype, that should be extracted from the linkset response not looking to the configuration
- the response is not empty, maybe we can check that the response contains the item title: this should be true regardless of which crosswalk we will use

This because if we improve the datacite xslt later on we will broken this test without a real reason

Reply:
Thank you for review!
Fixed